sdk/app_cpu1/common/drv/amds.c

```diff
@@ -70,6 +70,77 @@ int amds_get_data(uint8_t port, amds_channel_e channel, int32_t *out)
     }
 }
 
+/*
+ * This function gets the voltage in volts as a double for a given AMDS port, channel, and card type
```

Comment:
```suggestion
/* This function gets the voltage in volts as a double for a given AMDS port, channel, and card type
```

Other comments in this file start the text on the first line of the comment. Not sure which is more standard, but let's be consistent one way or the other.